### PR TITLE
Portfolio theme [op 52]

### PR DESCRIPTION
**Pull request by Rytiggy**

### CODE REVIEW

File: app/components/layer-image-text/style.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
-.image img {
-    width: 100%;
-    overflow: hidden;
+img {
+    width:500px;
```
Comment (caneruguz) [inline app/components/layer-image-text/style.scss]: I see why you did this, but this makes the image not responsive, especially on small windows. Can you test and check for alternatives.

File: app/components/add-layer/component.js [hunk 2]
```diff
@@ -37,6 +38,17 @@ export default Ember.Component.extend({
                                 "validation": null
                             }
                             ]
+                        },
+                        {
+                            "type": "alignment",
+                            "label": "Alignment",
+                            "value": "alignment",
+                            "options": [
+                            "left",
```
Comment (caneruguz) [inline app/components/add-layer/component.js]: Fix indentation please

File: app/components/edit-text/style.scss [hunk 3]
```diff
@@ -1,12 +1,42 @@
...
+    text-align: left;
+    border: none;
+    background: rgba(158, 158, 158, 0.08);
+    font-size: 30px;
+    width:100%;
+    border-radius: 2px;
+}
+.center {
+    text-align: center;
+    border: none;
```
Comment (caneruguz) [inline app/components/edit-text/style.scss]: You can combine all these repeating rules by doing this: 

```css
.left, .right, .center {
 ...
}

Comment (caneruguz) [inline app/components/edit-text/style.scss]: But semantically speaking, these are not related at all to alignment, maybe you need a different class there

File: app/components/edit-text/template.hbs [hunk 4]
```diff
@@ -1,8 +1,8 @@
 {{#if (eq type 'title')}}
     {{#if editMode}}
-        {{input class="edit-text-title"  maxlength=140 value=layer.settings.values.sectionTitle}}
+        {{input type="text" class=layer.settings.values.alignment maxlength=140 value=layer.settings.values.sectionTitle }}
```
Comment (caneruguz) [inline app/components/edit-text/template.hbs]: Somewhere else you took the input out of helper and just used <input>. The helper is really just a wrapper, you can use tags too.

File: app/components/layer-link/template.hbs [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {{yield}}
 
-{{edit-text showSettings=showSettings editMode=editMode layer=layer type="title"}}
+{{edit-text showSettings=showSettings editMode=editMode layer=layer type="title"  alignment=layer.settings.values.alignment}}
```
Comment (caneruguz) [inline app/components/layer-link/template.hbs]: This edit-text component is a bit problematic, I think its not quite necessary. We may look into refactoring it.

Comment (Rytiggy) [inline app/components/layer-link/template.hbs]: This could be a ticket to do in the future

File: app/components/edit-text/style.scss (hunk 3 above)
Comment (Rytiggy) [inline app/components/edit-text/style.scss]: These three classes are changing the text-align
